src/naive/time/mod.rs

```diff
@@ -190,6 +190,15 @@ pub struct NaiveTime {
     frac: u32,
 }
 
+#[cfg(feature = "arbitrary")]
+impl arbitrary::Arbitrary<'_> for NaiveTime {
+    fn arbitrary(u: &mut arbitrary::Unstructured) -> arbitrary::Result<NaiveTime> {
+        let secs = u.int_in_range(0..=86_400)?;
```

Comment:
If I understand correctly these ranges should not be inclusive... a day has up to 86399 seconds, and `frac` is supposed to be smaller than `2_000_000_000`.

See https://github.com/chronotope/chrono/blob/1281ce01e56caf39a1c902086707efd796f6f2f6/src/naive/time/mod.rs#L391.